apollo-router/src/plugins/traffic_shaping/mod.rs

```diff
@@ -447,7 +447,7 @@ impl TrafficShaping {
                     req
                 }))
         } else {
-            Either::B(service)
+            Either::Right(service)
```

Comment:
... the variants used to be named ... `A` and `B`? :scream: that's ... very descriptive :scream:

Reply:
That's an underlying crate name change.